code/parachain/frame/composable-traits/src/staking/lock.rs

```diff
@@ -16,13 +16,52 @@ use crate::time::{DurationSeconds, Timestamp};
 #[scale_info(skip_type_params(MaxDurationPresets))]
 pub struct LockConfig<MaxDurationPresets: Get<u32>> {
 	/// The possible locking duration.
-	pub duration_presets:
-		BoundedBTreeMap<DurationSeconds, Validated<FixedU64, GeOne>, MaxDurationPresets>,
+	pub duration_multipliers: DurationMultipliers<MaxDurationPresets>,
 	/// The penalty applied if a staker unstake before the end date.
 	/// In case of zero penalty, you cannot unlock before it duration ends.
 	pub unlock_penalty: Perbill,
 }
 
+/// Represents both valid staking lock durations and ther associated multiplier bonus.
+#[derive(
+	DebugNoBound, PartialEqNoBound, EqNoBound, CloneNoBound, MaxEncodedLen, Encode, Decode, TypeInfo,
+)]
+#[scale_info(skip_type_params(MaxDurationPresets))]
+pub enum DurationMultipliers<MaxDurationPresets: Get<u32>> {
+	/// Fixed duration multipliers mapped to their respective multipliers.
+	// TODO(benluelo): Wrap this in `Validated` to ensure that at least one preset is provided?
```

Comment:
We can avoid TODO for this types of comments, it's confusing to readers. Let's just make it "Possible Improvement" or similar

Reply:
Besides that, we can also have a default for a pool or do validation both are fine by me

Reply:
I don't think it's confusing for readers; this TODO will be addressed in a followup PR anyways. I wanted to keep this PR as simple as possible, not introducing any changes other than wrapping the presets in the enum

Reply:
Work generally should not get increased with TODOs as that is unpredictable to understand when something is actually "done" and ready to release. We can create clickup tasks and prioritize if needed.